.github/workflows/weekly_system_test.yml

```diff
@@ -0,0 +1,42 @@
+name: Weekly System Test
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 8 * * 0"
+
+jobs:
+  system-checks:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        python-version: [3.6, 3.7, 3.8, 3.9]
```

Comment:
I think we can drop support for Python 3.6 since it is EOL.

Reply:
I have created a [separate branch](https://github.com/aisingapore/PeekingDuck/tree/build-support-py38-py310) for migrating to Py38-Py310.

I was thinking for now `dev` branch should still behave as if we're supporting Py36-Py39 and all work related to bumping the Python version will happen on that branch so the changes will look cleaner.

Reply:
Ok, this will do too.